adb_android/adb_android.py

```diff
@@ -88,7 +88,7 @@ def wait_for_device():
 
 def sync():
     """
-    Perform a sync with device and host after a series of commands executed
+    Perform a sync with device and host
```

Comment:
Let's be more precise here by specifying what exactly will be synchronized. Android docs say 

> "copy host->device only if changed"